.github/workflows/python-publish.yml

```diff
@@ -66,4 +66,4 @@ jobs:
       env:
         DISCORD_WEBHOOK: ${{ secrets.DISCORD_WEBHOOK }}
       with:
-        args: 'Launchable CLI {{ EVENT_PAYLOAD.release.tag_name }} is released! {{ EVENT_PAYLOAD.release.html_url }}'
+        args: 'Launchable CLI {{ needs.tagpr.outputs.tag }} is released!'
```

Comment:
I think we can replace from `{{ EVENT_PAYLOAD.release.html_url }}` to `https://github.com/launchableinc/cli/releases/tag/{{ needs.tagpr.outputs.tag }}`.
WDYT??

Reply:
[0feb57a](https://github.com/launchableinc/cli/pull/605/commits/0feb57ad7f045d13a008fe449bb6799df68f0919)